paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTest.kt

```diff
@@ -174,7 +174,7 @@ internal class PaymentOptionsViewModelTest {
         )
 
         viewModel.currentScreen.test {
-            assertThat(awaitItem()).isEqualTo(SelectSavedPaymentMethods())
+            assertThat(awaitItem()).isInstanceOf(SelectSavedPaymentMethods::class.java)
```

Comment:
You can also write these as `assertThat(awaitItem()).isInstanceOf<SelectSavedPaymentMethods>()` which gets rid of the `::class.java` uglies.

Reply:
will follow up with this update -- I think I did the same thing in the AddPaymentMethod interactor refactor, so I can update those spots too